base/docs/helpdb/Base.jl

```diff
@@ -2265,11 +2265,11 @@ sortperm
 """
     mod2pi(x)
 
-Modulus after division by 2pi, returning in the range \[0,2pi).
+Modulus after division by ``2π``, returning in the range ``\[0,2π)``.
 
 This function computes a floating point representation of the modulus after division by
-numerically exact 2pi, and is therefore not exactly the same as mod(x,2pi), which would
-compute the modulus of `x` relative to division by the floating-point number 2pi.
+numerically exact ``2π``, and is therefore not exactly the same as `mod(x,2π)`, which would
+compute the modulus of `x` relative to division by the floating-point number ``2π``.
```

Comment:
Since the `2π` at the end of this doc is referring to the floating point value rather than mathematical 2π perhaps it should just be single backticks:

```
... floating-point number `2π`.
```